contracts/adapters/swap/astroport/tests/test_execute_swap.rs

```diff
@@ -1,10 +1,15 @@
+use std::vec;
```

Comment:
nit: I think can remove all places where u import this, don't think necessary

Reply:
whoops, I think this was my autocomplete